docs/setup-robusta/alertsui.rst

```diff
@@ -0,0 +1,74 @@
+Alerts UI
+######################
+
+With Robusta Alerts UI, you can create and customize Prometheus Rules using templates, without needing to know PromQL.
+
+.. image:: /images/managed-alerts-ui.png
+
+
+
+This guide covers enabling managed alerts with Robusta UI. This involves two steps: One to enable the Alert Rules on the UI and another to disable default Prometheus alerts installed by Kube Prometheus Stack.
+
+.. note::
+
+    Alerts UI works only with Prometheus installed by Robusta. :ref:`Embedded Prometheus Stack`
```

Comment:
I suggest rephrasing as: 

`To use the Alerts UI, you must use Robusta's embedded Prometheus stack.`